templates/details/libraries/libraries_layout.html

```diff
@@ -81,7 +81,23 @@ <h3>{{ docstrings["name"] }}</h3>
           <li class="p-inline-list__item">
             <ul class="p-inline-list has-vertical-separator">
               <li class="p-inline-list__item">
-                <a href="#" class="is-always-color-link"><i class="p-icon--download">Download</i> <span>Download</span></a>
+                <div class="p-modal-tooltip">
+                  <button class="p-button--base is-always-color-link has-icon u-no-margin--bottom" id="download-modal-button" aria-controls="download-modal">
+                    <i class="p-icon--download"></i>
+                    <span>Download</span>
+                  </button>
+                  <div class="p-modal-tooltip__wrapper u-hide" id="download-modal">
+                    <div class="p-modal-tooltip__body" aria-controlledby="download-modal-button">
+                      <p class="u-no-padding--top"><i class="p-icon--code"></i> Fetch library</p>
+                      <pre class="u-no-margin--bottom">charmcraft fetch-libs charms/{{library["charm-name"]}}/v{{library["api"]}}/db.py</pre>
```

Comment:
This should stay: `charmcraft fetch-libs test`

The name of the library should be given by the API no need to build it.

Reply:
done